docs/src/Toast.doc.js

```diff
@@ -327,6 +327,7 @@ card(
       <Image
         key="image-key"
         alt="Saved to home decor board"
+        color="transparent"
```

Comment:
If it's a default defining `color="transparent"` could be redundant

Reply:
Flow complains if this is removed. This is what I was referring to in the summary; Flow handles default arg values and defaultProps differently, and doesn't understand that this has a default value.